pkgs/tools/backup/diskrsync/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildGoPackage, fetchFromGitHub, stdenv }:
+
+buildGoPackage rec {
+
+  name = "${pname}-${version}";
+  pname = "diskrsync";
+  version = "20170927";
```

Comment:
Could you please use the name format `NAME-unstable-YYYY-MM-DD` as specified in https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Fixed.